src/verify.py

```diff
@@ -0,0 +1,13 @@
+array = [2,4,6,8,10,12,14,16,18,20]
+
+def verify(number : int) -> bool:
```

Comment:
Sarebbe meglio usare un nome più descrittivi per la funzione (es: is_element_in_list oppure list_contains), in modo che, semplicemente leggendo la firma della funziona, si intuisca cosa si aspetta in input, cosa ci aspettiamo che faccia e cosa eventualmente ritorna come output.
In questo caso potrebbe sembrare superfluo perché la funzione è banale, tuttavia è una buona abitudine che è meglio acquisire il prima possibile :)

Reply:
> Sarebbe meglio usare un nome più descrittivi per la funzione (es: is_element_in_list oppure list_contains), in modo che, semplicemente leggendo la firma della funziona, si intuisca cosa si aspetta in input, cosa ci aspettiamo che faccia e cosa eventualmente ritorna come output. In questo caso potrebbe sembrare superfluo perché la funzione è banale, tuttavia è una buona abitudine che è meglio acquisire il prima possibile :)

done